From 33a7dc7e66c9ec3c9f88a0940fb562fb60b37a6e Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Wed, 27 Feb 2013 13:38:52 +0800
Subject: [PATCH 166/310] ENGR00251842-2 imx6dq: merge AOSP patches from
 JB4.2.2

merge patch from Todd Poynor <toddpoynor@google.com>
rootdir: init.rc: set perms for cpufreq governor boostpulse_duration

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 imx6/etc/init.rc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index b09fcb6..45d7c2f 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -424,6 +424,8 @@ on property:sys.interactive="active"
     chmod 0660 /sys/devices/system/cpu/cpufreq/interactive/boostpulse
     chown system root /sys/devices/system/cpu/cpufreq/interactive/input_boost
     chmod 0660 /sys/devices/system/cpu/cpufreq/interactive/input_boost
+	chown system system /sys/devices/system/cpu/cpufreq/interactive/boostpulse_duration
+	chmod 0660 /sys/devices/system/cpu/cpufreq/interactive/boostpulse_duration
 
 
 on nonencrypted
-- 
1.8.0

